projects/media-viewer/src/lib/media-viewer.component.ts

```diff
@@ -90,6 +98,7 @@ export class MediaViewerComponent implements OnChanges, OnDestroy, AfterContentI
 
   ngOnDestroy() {
     this.commentState.unsubscribe();
```

Comment:
the pattern we're following in other components is to add the subscriptions anonymously to a list and then unsubscribe them all